public/js/sticky-sidebar.js

```diff
@@ -0,0 +1,38 @@
+/* global $ */
```

Comment:
As this is being `require`d in the main javascript file, it might be better to export a the function  on line 7, thereby letting the consumer of this module decide when to initialise the sticky behaviour.

E.G.
```js
module.exports = function() {
    var stickySidebar = $('.js-sticky-sidebar);
...
};
```

This would also make the module a bit simpler to test as we wouldn't have to fiddle around with firing `DOMConentLoaded` events to trigger it.